### ISSUE TITLE
Error in nuclide-fuzzy-filename-provider (Nuclide 0.167)

### ISSUE THREAD

**Issue by ffxsam** [facebook/nuclide#699]
[Enter steps to reproduce below:]

1. Open Atom+Nuclide
2. Boom.

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [nuclide-fuzzy-filename-provider](https://github.com/facebook/nuclide) package, v0.0.0

### Stack Trace

Failed to activate the nuclide-fuzzy-filename-provider package

```
At Must call setUseLocalRpc before getService

AssertionError: Must call setUseLocalRpc before getService
    at getlocalService (/Users/samh/.atom/packages/nuclide/pkg/nuclide-remote-connection/lib/service-manager.js:71:39)
    at getService (/Users/samh/.atom/packages/nuclide/pkg/nuclide-remote-connection/lib/service-manager.js:114:12)
    at getServiceByNuclideUri (/Users/samh/.atom/packages/nuclide/pkg/nuclide-remote-connection/lib/service-manager.js:99:10)
    at /Users/samh/.atom/packages/nuclide/pkg/nuclide-fuzzy-filename-provider/lib/main.js:107:104
    at Array.forEach (native)
    at initSearch (/Users/samh/.atom/packages/nuclide/pkg/nuclide-fuzzy-filename-provider/lib/main.js:102:16)
    at Activation.activate (/Users/samh/.atom/packages/nuclide/pkg/nuclide-fuzzy-filename-provider/lib/main.js:73:7)
    at getActivation (/Users/samh/.atom/packages/nuclide/pkg/nuclide-fuzzy-filename-provider/lib/main.js:90:16)
    at Object.activate (/Users/samh/.atom/packages/nuclide/pkg/nuclide-fuzzy-filename-provider/lib/main.js:152:3)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:183:20)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:538:21)
    at /Users/samh/.atom/packages/nuclide/lib/main.js:318:21
    at Array.forEach (native)
    at Object.activate (/Users/samh/.atom/packages/nuclide/lib/main.js:315:25)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:183:20)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:538:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:519:29
    at Config.module.exports.Config.transactAsync (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:514:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:497:46)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:765:28
```

### Commands

```
     -0:00.9.0 nuclide-home:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-solarized-dark-syntax.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "tree-view",
      "linter"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "node_modules",
      "local",
      ".idea"
    ],
    "projectHome": "/Users/samh/Sites",
    "themes": [
      "one-dark-ui",
      "solarized-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-modules, v1.6.1 (inactive)
expose, v0.12.1 (inactive)
git-time-machine, v1.5.3 (inactive)
goto-definition, v1.1.19 (inactive)
haskell-grammar, v0.4.0 (inactive)
highlight-selected, v0.11.2 (inactive)
js-refactor, v0.7.3 (inactive)
language-babel, v2.37.0 (inactive)
language-ini, v1.16.0 (inactive)
language-ocaml, v1.1.2 (inactive)
language-swift, v0.4.0 (inactive)
language-thrift, v1.0.2 (inactive)
linter, v1.11.16 (inactive)
linter-eslint, v7.3.0 (inactive)
merge-conflicts, v1.4.4 (inactive)
minimap, v4.25.0 (inactive)
nuclide, v0.167.0 (inactive)
refactor, v0.9.2 (inactive)
sort-lines, v0.14.0 (inactive)
todo-show, v1.7.0 (inactive)
tool-bar, v1.0.1 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (inactive)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (inactive)
archive-view, v0.61.1 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.11.2 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.1 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.41.0 (inactive)
bracket-matcher, v0.82.1 (inactive)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.39.0 (inactive)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (inactive)
image-view, v0.58.2 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (inactive)
metrics, v0.53.1 (inactive)
notifications, v0.65.0 (inactive)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (inactive)
snippets, v1.0.2 (inactive)
spell-check, v0.67.1 (inactive)
status-bar, v1.4.1 (inactive)
styleguide, v0.47.0 (inactive)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.208.2 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.34.0 (inactive)
whitespace, v0.33.0 (inactive)
wrap-guide, v0.38.1 (inactive)
language-c, v0.52.1 (inactive)
language-clojure, v0.21.0 (inactive)
language-coffee-script, v0.47.2 (inactive)
language-csharp, v0.12.1 (inactive)
language-css, v0.37.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.42.1 (inactive)
language-html, v0.45.1 (inactive)
language-hyperlink, v0.16.0 (inactive)
language-java, v0.23.0 (inactive)
language-javascript, v0.119.0 (inactive)
language-json, v0.18.2 (inactive)
language-less, v0.29.5 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.35.0 (inactive)
language-php, v0.37.2 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.0 (inactive)
language-ruby, v0.69.0 (inactive)
language-ruby-on-rails, v0.25.0 (inactive)
language-sass, v0.56.0 (inactive)
language-shellscript, v0.22.4 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.23.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.28.0 (inactive)
language-toml, v0.18.0 (inactive)
language-xml, v0.34.9 (inactive)
language-yaml, v0.26.0 (inactive)

# Dev
No dev packages
```

**Comment by zertosh** [comment]
Sorry, we're aware of the issue and working on a fix. Tracking this in #698.

**Comment by zertosh** [comment]
Just published `nuclide@v0.167.1`. If you update to it, and reload Atom, everything should be working again.
